ui/app/components/app/permission-page-container/permission-page-container-content/permission-page-container-content.component.js

```diff
@@ -0,0 +1,144 @@
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import Identicon from '../../../ui/identicon'
+import IconWithFallBack from '../../../ui/icon-with-fallback'
+import classnames from 'classnames'
+
+export default class PermissionPageContainerContent extends PureComponent {
+
+  static propTypes = {
+    requestMetadata: PropTypes.object.isRequired,
+    domainMetadata: PropTypes.object.isRequired,
+    selectedPermissions: PropTypes.object.isRequired,
+    permissionsDescriptions: PropTypes.object.isRequired,
+    onPermissionToggle: PropTypes.func.isRequired,
```

Comment:
I'm assuming this _was_ meant to be used somewhere? Otherwise permissions can't be toggled.

If permissions aren't to be toggle-able anymore, then the parent component can be simplified somewhat. I'm guessing that is still desired though.

Reply:
Re-added the use of this prop in c936649fc